models/resnet.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, in_channels, out_channels):
 
     def forward(self, x):
         # x -> conv1 -> bn1 -> ReLU -> maxpool
-        x = F.leaky_relu(self.bn1(self.conv1(x)))
+        x = F.leaky_relu(self.bn1(self.conv1(x.float())))

```

Comment:
this could cause an issue in pytorch if the type is not transferred properly.

Reply:
We will not convert type in the model
If we need to change data type, we will do it at the dataloader.
This is so that we don't need to keep changing it in the future model files